content/dstu2/allergy-intolerance.md

```diff
@@ -0,0 +1,128 @@
+---
+title: AllergyIntolerance | FHIR DSTU 2 API
+---
+
+# AllergyIntolerance
+
+* TOC
+{:toc}
+
+## Search
+
+Search for AllergyIntolerances that meet supplied query parameters:
+
+    GET /AllergyIntolerance?:parameters
+
+### Parameters
+
+ Name    | Required? | Type                                                           | Description
+---------|-----------|----------------------------------------------------------------|---------------------------------------------
+`patient`|     Y     |[`reference`](http://hl7.org/fhir/DSTU2/search.html#reference)| Who the sensitivity is for. Example: `12345`
+`status` |     N     |[`token`](http://hl7.org/fhir/DSTU2/search.html#token)        | [Certainty of the allergy or intolerance](http://hl7.org/fhir/DSTU2/valueset-allergy-intolerance-status.html). Example: `confirmed`
+
+### Response
+
+<%= headers 200 %>
+<%= json(:dstu2_allergy_intolerance_bundle) %>
```

Comment:
JSON rendered this way is used in create and update as the request body style instead of the response body.  We might want to provide a visual queue in addition to the section headers to differentiate request vs response.  If we do change this it will require changes in the other resources too